src/states/Midroll.js

```diff
@@ -17,7 +17,6 @@ export default class Midroll extends AdState {
     const player = this.player;
 
     if (!this.inAdBreak() && !this.isContentResuming()) {
-      player.ads.debug('startLinearAdMode (Midroll)');
```

Comment:
why remove this log?

Reply:
I had added it in a relatively recent commit and found it's basically redundant with the "Starting ad break" log.